SECURITY_CONTACTS

```diff
@@ -0,0 +1,11 @@
+# Defined below are the security contacts for this repo.
+#
+# They are the contact point for the Product Security Team to reach out
+# to for triaging and handling of incoming issues.
+#
+# The below names agree to abide by the
+# [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy)
+# and will be removed and replaced if they violate that agreement.
+
+bob
+alice
```

Comment:
There are unlikely to be real security issues with the template repo, but do we want to choose a few names from [here][1] to stick wigh the "using real usernames" precedent?  I assume the steering committee are all on board with the embargo policy?

[1]: https://github.com/kubernetes/kubernetes-template-project/blob/master/OWNERS_ALIASES

Reply:
This repo is designed to be a template. Having these names indicates the template hasn’t been filled out yet.

Reply:
> This repo is designed to be a template. Having these names indicates the template hasn’t been filled out yet.

So `OWNERS` has, via `OWNERS_ALIASES`, real usernames, because we need those to maintain the template.  But this file can get dummy usernames because we'll never never have real security issues for the template?  That sounds reasonable.

I'd still prefer we use non-dummy values here.  Being the official template security contact should not be a heavy burden, because we expect no security issues here.  Project based on this template will need to replace these values whether they are dummies here or not.  And in the event that a project somehow forgets to update the values, the steering committee can be relied on to not leak anything, while `alice` and `bob` are real GitHub accounts without current Kubernetes associations.

Also, are GitHub notifications the recommended disclosure channel?  Perhaps these should be email addresses with (optional?) OpenPGP or otherwise public keys?

Reply:
This file is not intended to be used by the public. As noted in the files itself, this is intended to be used by our PST to identify the right people to bring in to triage issues.

The public disclosure path is here: https://kubernetes.io/security/

Reply:
> As noted in the files itself, this is intended to be used by our PST to identify the right people to bring in to triage issues.

How will they translate GitHub usernames into contact requests?  With an initial "please join us on ${secure_channel}" request be in a GitHub issue?  Will they extract a usable email address from the user's public commit history?  Or is the contact information intended to be conveyed via a non-GitHub channel?

> The public disclosure path is here: https://kubernetes.io/security/

I think we should mention that in the comment here as well, to avoid having folks reading this file interpret it as the public channel.

Reply:
Contact information be handled via a non-Github channel. Security contacts will be well establish community members.

Adding a line stating the public disclosure path is not a bad idea. WDYT, @jessfraz?